draft-ietf-quic-applicability.md

```diff
@@ -299,6 +299,18 @@ not behind a load balancer, and therefore it is unlikely that the addresses
 will be re-bound, may indicate to the transport that is wishes to not expose a
 Connection ID.
 
+Server-generated Connection IDs must not encode any information other than that
```

Comment:
Hey, wait... this entire blurb is already in the following section it looks like. Right?

Reply:
oh, indeed. we already copied this over, but didn't remove it from -manageability.